src/DI/Nette/FioApiExtension.php

```diff
@@ -58,4 +46,16 @@ public function loadConfiguration(): void
 			]);
 	}
 
+	public function getConfigSchema(): Nette\Schema\Schema
```

Comment:
Presun prosim schema uplne nahoru. Je to takovy dobry zvyk, mit to nahore, kdyz si to clovek otevre, aby to videl.

Reply:
ok